### ISSUE TITLE
decorator limiter would not limiter the time between two tick from tree

### ISSUE THREAD

**Issue by linhaiwei123** [behavior3/behavior3js#25]
the tree is as below
![image](https://cloud.githubusercontent.com/assets/17567475/24079568/edddd120-0cc5-11e7-9ba7-b6d3cd1a6589.png)

and i do enter the limiter but when i tick the tree in the update again  the time of the limiter would reset to zero again.

**Comment by linhaiwei123** [comment]
i found the bug here. in limiter we should return the b3.RUNNING instead of b3.SUCCESS or b3.FAILURE
cause the b3.SUCCESS or b3.FAILURE would exit the running state and  reset the time in open when we tick again

**Comment by linhaiwei123** [comment]
yep i found that return b3.RUNNING in limiter still not work well.  the only way to fix it is to check whether the key i is set or not. and set it in the first time we enter the open callback

**Comment by danielepolencic** [comment]
@linhaiwei123 would you be able to submit a PR with the failing test or a simple way to reproduce the bug?

**Comment by wqhk** [comment]
open: function(tick) {
    tick.blackboard.set('i', 0, tick.tree.id, this.id);
  },
The Decorator of Limiter will reset "i"  to zero when child status == b3.SUCCESS || status == b3.FAILURE。  +1 ，reset 0， +1 ，reset 0……

**Comment by danielepolencic** [comment]
@wqhk would you be able to submit a PR with a failing test?

**Comment by wqhk** [comment]
@danielepolencic  I submit a PR.  Learn BT this week, maybe misunderstand the usage of Limiter =。=
